embark.el

```diff
@@ -3194,13 +3197,13 @@ buffer."
                (cmd embark--command)
                (name (embark--descriptive-buffer-name 'export))
                (rerun (embark--rerun-function #'embark-export))
-               (buffer (save-excursion
-                         (funcall exporter candidates)
+               (buffer-display embark--buffer-display)
+               (buffer (with-current-buffer (funcall exporter candidates)
                          (rename-buffer name t)
                          (current-buffer))))
           (embark--quit-and-run
            (lambda ()
-             (pop-to-buffer buffer)
```

Comment:
Looking at this code, I wonder why this call is here, since it somehow contradicts the original implementation where the exporter also creates windows. The reason is probably that some exporter created windows and some do not.

Reply:
Yes. The original motivation of this PR is to achieve consistency here. For now we did renaming the buffer twice.

Reply:
> Yes. The original motivation of this PR is to achieve consistency here.

That's good. 

> For now we did renaming the buffer twice.

Yes, seems to make sense to clean that up, but it seems there are more exporters which may need similar changes.